packages/paste-core/components/date-picker/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@twilio-paste/date-picker",
+  "version": "0.0.0",
+  "category": "user input",
+  "status": "alpha",
+  "description": "A form component used to select a date.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "dependencies": {
+    "date-fns": "^2.21.3"
```

Comment:
We can safely remove `^` as you brought up because it will reduce margin for error in consumer installs.